src/Core/Util/ArabicToLatinDigitConverter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Util;
+
+/**
+ * Utility class to convert arabic/persian digits to latin digits
+ */
+class ArabicToLatinDigitConverter
+{
+    private static $translationTable = [
+        // arabic numbers
+        '٠' => '0',
+        '١' => '1',
+        '٢' => '2',
+        '٣' => '3',
+        '٤' => '4',
+        '٥' => '5',
+        '٦' => '6',
+        '٧' => '7',
+        '٨' => '8',
+        '٩' => '9',
+        // persian numbers (NOT the same UTF codes!)
+        '۰' => '0',
+        '۱' => '1',
+        '۲' => '2',
+        '۳' => '3',
+        '۴' => '4',
+        '۵' => '5',
+        '۶' => '6',
+        '۷' => '7',
+        '۸' => '8',
+        '۹' => '9',
+    ];
+
+    /**
+     * Convert from arabic/persian digits to latin digits
+     *
+     * @param string $str
+     *
+     * @return string
+     */
+    public function convert(string $str): string
+    {
+        return strtr($str, self::$translationTable);
+    }
+
+    /**
+     * Convert from latin digits to arabic digits
+     *
+     * @param string $str
+     *
+     * @return string
+     */
+    public function reverseConvert(string $str): string
```

Comment:
What about reverse into persian? I know it looks the same and only UTF codes change, but still Why favor arabic over persian?

Reply:
@jolelievre I had to make a choice 😄 I can separate in 2 methods (`latinToArabic()` and `latinToPersan()`) in this class and only use one in `DatePickerType` though! And because it will be replaced by javascript in the end that would be ok. WDYT?

Reply:
Either two method or a parameter to chose the translation table But yeah I get that one will be automatically used in the `DatePickerType` But at least this class will be more versatile from a stand alone point of view